src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.FruitStorage;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void addFruit(String fruit, int amount) {
+        FruitStorage.fruitRepository.put(fruit, amount);
+    }
+
+    @Override
+    public int get(String fruit) {
+        return FruitStorage.fruitRepository.get(fruit);

```

Comment:
There is a potential `NullPointerException` if the `fruitRepository` map does not contain the given fruit. Use `getOrDefault` method to avoid this exception and return a default value like 0.